### PR TITLE
link attendee information, fix date in temporal snippet

### PR DESCRIPTION
**Pull request by bakkot**
We should make a checklist for copying over the templates, given how many of these little details need to be gotten right.

### CODE REVIEW

File: 2023/11.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ gantt
 For meeting times in your timezone, visit [Temporal docs](https://tc39.es/proposal-temporal/docs/) and run the code below in the devtools console.
 
 ```js
-Temporal.ZonedDateTime.from('2023-11-17T10:00[America/Los_Angeles]')
```
Comment (michaelficarra) [inline 2023/11.md:27]: As written, this gives you the advancement deadline. I think that's fine.

Comment (bakkot) [reply 2023/11.md:27]: That would be fine if it said "for _the advancement deadline_ in your timezone", but it claims to be the meeting time.

**Comment by ctcpip** [comment]
> We should make a checklist for copying over the templates, given how many of these little details need to be gotten right.

the agenda and the reflector issue are separate activities.  when there's a `.race()`, and the agenda appears first, the reflector link isn't available.

**Comment by bakkot** [comment]
Sure, but it's not just that one; there's a bunch of dates and etc which need to be updated.

**Comment by ctcpip** [comment]
clipgenda

**Review by ctcpip — APPROVED** [review]

**Pull request merged**